app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,14 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      # TODO: why doesn't this correctly output html without `html_safe`?
```

Comment:
I think the issue is that the `flash` object persists the string in the session, so it loses the HTML safety that Activesupport monkeypatches in to the object. Maybe we could look into persisting the translation key in the flash instead of the message, and then looking that up when we render? (it would be a bigger change but it's much safer than HTML-safe-ing arbitrary strings from the session IMO)

ex

here:
```
flash[:success] = 'foo.bar.baz'
```

in `_messages.html.slim`
```slim
- unless flash.empty?
  - flash.each do |name, msg|
    - if msg.present? && msg.is_a?(String)
      div class="alert alert-#{name}" role='alert'
        = t(msg)

```